css/selectors/left.json

```diff
@@ -7,10 +7,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:left",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": false
```

Comment:
Those links are referring to the `left` property rather than the selector.  From what I can tell, `:left` and `:right` had no effect on the pages in Chrome 74 (when trying to print).